### DISCUSSION TITLE
ossec-authd ..can't find it

### DISCUSSION THREAD

**Discussion by peper66** [Security-Onion-Solutions/securityonion#3878] [Unsupported Versions]
I'm new to Security Onion and want to automated wazuh client for windows deployement
 there is no ossec-authd in the /var/ossec/bin

Any help ?

https://documentation.wazuh.com/3.13/user-manual/reference/daemons/ossec-authd.html

**Comment by lan94** [comment]
Did you looked on the host or in the container? On the Host you should have installed the wazuh agent without ossec-authd. To get it, try to look into your wazuh container like:

`sudo docker exec so-wazuh ls /var/ossec/bin`

There you should find this file.
